lib/js-api/src/api/content-rest-api/docs/SizeDetails.md

```diff
@@ -0,0 +1,12 @@
+# Size Details
+
+## Properties
+
+ Name              | Type   | Description                                                                 | Notes             
+-------------------|--------|-----------------------------------------------------------------------------|-------------------
+ **id**            | string |                                                                             | [default to null] 
```

Comment:
could you please fill description column for this field? Probably everyone knows what id is but just to keep consistency with other fields which have filled that column

Reply:
Filled description column